hypergraph/concepts/03-integrationnet.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Integration net
+slug: /integrationnet
+sidebar_label: Integration net
+hide_table_of_contents: false
+---
+<intro-end />
+
+Constellation provides developers with a full featured integration net to test applications and metagraphs before they're ready for the production environment. 
+The integration net has all of the same features as mainnet and can therefore be used to validate application integrations in a realistic way. 
```

Comment:
We need a note somewhere in here that indicates that there's a seedlist and how to apply to the seedlist.

Reply:
the info section already points out to the setup a metagraph guide, that has that information, do you want to also have in this page?